### PR TITLE
Fix bug for Zeroconf

### PR DESCRIPTION
**Pull request by ziransun**
Jira Issue: WP-98

### CODE REVIEW

File: webinos/pzp/lib/pzp_sessionHandling.js [hunk 1]
```diff
@@ -409,27 +386,27 @@ Pzp.prototype.connect = function (conn_key, conn_csr, code, callback) {
...
+      if (self.webServerState !== global.states[2]) {
+        pzpWebSocket.startPzpWebSocketServer(self, self.inputConfig, function() {
+          self.rpcHandler.setSessionId(self.sessionId);
+          setupMessageHandler(self);
+          self.connectedApp();
+          log.info("started pzp");
+          self.webServerState = global.states[2];
+        });
+      }
+   if(self.pzptlsServerState === global.states[0])
```
Comment (habibvirji) [inline webinos/pzp/lib/pzp_sessionHandling.js]: please check the indentation

File: webinos/pzp/lib/pzp_sessionHandling.js [hunk 2]
```diff
@@ -409,27 +386,27 @@ Pzp.prototype.connect = function (conn_key, conn_csr, code, callback) {
...
-        } 
-      } else {
+      }       
+     
+     // Special case if started in hub disconnected mode
+      if (self.webServerState !== global.states[2]) {
+        pzpWebSocket.startPzpWebSocketServer(self, self.inputConfig, function() {
+          self.rpcHandler.setSessionId(self.sessionId);
+          setupMessageHandler(self);
+          self.connectedApp();
```
Comment (habibvirji) [inline webinos/pzp/lib/pzp_sessionHandling.js]: why is update function not used?

File: webinos/pzp/lib/pzp_sessionHandling.js [hunk 3]
```diff
@@ -483,7 +460,31 @@ Pzp.prototype.connectedApp = function(connection) {
       }
     }
   }
- 
+  
+  /*
```
Comment (habibvirji) [inline webinos/pzp/lib/pzp_sessionHandling.js]: as discussed previously connected app might not be appropriate place as this called when new connection is made from browser

File: webinos/pzp/lib/pzp_sessionHandling.js (hunk 2 above)
Comment (ziransun) [inline webinos/pzp/lib/pzp_sessionHandling.js]: callback is not required here which has been included in update function

File: webinos/pzp/lib/pzp_sessionHandling.js (hunk 3 above)
Comment (ziransun) [inline webinos/pzp/lib/pzp_sessionHandling.js]: Put advertisement into update function doesn't work.

**Pull request merged**